src/core/CoolWifi.cpp

```diff
@@ -96,7 +87,7 @@ wl_status_t CoolWifi::connectWifiMulti() {
   return (status);
 }
 
-wl_status_t CoolWifi::connectAP() {
+wl_status_t CoolWifi::connectAP(String ssidTmp[], String passTmp[]) {
```

Comment:
This function is not being called anywhere anymore, which is weird. How are we supposed to start the access point now ? Well, it's broken, isn't it ?

During a long time, the Facebook developers mantra was "go fast and break things". While this is cool for testing purposes, you are not expected to submit pull requests that break _essential_ things such as WiFi configuration. Again, sorry, but you are doing it wrong.

Reply:
i see it, thanks!